### PR TITLE
Adds ability to upload documents to adoption application

### PR DESCRIPTION
**Pull request by TheDudeWithTheThing**
some clean up as well

### CODE REVIEW

File: app/controllers/adopters_controller.rb [hunk 1]
```diff
@@ -26,29 +26,29 @@
 #
 
 class AdoptersController < ApplicationController
-  include SessionsHelper
```
Comment (TheDudeWithTheThing) [inline app/controllers/adopters_controller.rb]: this is already in `ApplicationController`

File: app/controllers/adopters_controller.rb [hunk 2]
```diff
@@ -26,29 +26,29 @@
...
-
-  before_filter :authenticate, except: [:new, :create, :check_email]
-  before_filter :edit_my_adopters_user, only: [:index, :show, :edit, :update]
-  before_filter :edit_all_adopters_user, only: [:index, :show, :edit, :update]
-  before_filter :admin_user, only: [:destroy]
-  before_filter :load_adopter, only: %i(show update)
+  before_action :authenticate, except: %i(new create check_email)
```
Comment (TheDudeWithTheThing) [inline app/controllers/adopters_controller.rb]: `before_action` is the new `before_filter`

**Comment by markottaviani** [comment]
Good with this one too!

**Comment by TheDudeWithTheThing** [comment]
👍

**Comment by TheDudeWithTheThing** [comment]
:shipit: 🚀

**Pull request merged**